### DISCUSSION TITLE
control master device from follower

### DISCUSSION THREAD

**Discussion by engramyhagag** [NightscoutFoundation/xDrip#2679] [General]
Hi 
can i control (as follower) my son device (the master) to make reset and other setting remotely

**Comment by Navid200** [comment]
If you are using xDrip sync to follow, not Nightscout, there is a setting under "xDrip sync settings" titled "Accept follower actions".  If you have that enabled on the master, there are some actions you can use from the follower.  For example, you will be able to stop the sensor from the follower.
